<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1642</width>
    <height>826</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(0, 0, 0);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>-20</y>
      <width>1541</width>
      <height>761</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">QTabWidget::pane { 
	border-style: solid;
	border-color: #3ACCCC;
	border-width: 0 0px 3px 0px; 
}</string>
    </property>
    <property name="tabPosition">
     <enum>QTabWidget::South</enum>
    </property>
    <property name="tabShape">
     <enum>QTabWidget::Triangular</enum>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <property name="iconSize">
     <size>
      <width>250</width>
      <height>150</height>
     </size>
    </property>
    <property name="elideMode">
     <enum>Qt::ElideMiddle</enum>
    </property>
    <widget class="QWidget" name="tab_5">
     <attribute name="icon">
      <iconset>
       <normalon>:/Media/icons/blackSquare.png</normalon>
      </iconset>
     </attribute>
     <attribute name="title">
      <string/>
     </attribute>
    </widget>
    <widget class="QWidget" name="tab_0">
     <attribute name="icon">
      <iconset resource="../Data.qrc">
       <normaloff>:/Media/icons/home.png</normaloff>
       <normalon>:/Media/icons/home.gif</normalon>:/Media/icons/home.png</iconset>
     </attribute>
     <attribute name="title">
      <string/>
     </attribute>
     <widget class="QWidget" name="horizontalLayoutWidget_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>220</y>
        <width>1351</width>
        <height>225</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <layout class="QGridLayout" name="gridLayout">
         <property name="spacing">
          <number>0</number>
         </property>
         <item row="0" column="1">
          <widget class="QLabel" name="label_am_pm">
           <property name="maximumSize">
            <size>
             <width>50</width>
             <height>100</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">color: #ffffff;
font: 600 18pt &quot;Yu Gothic UI Semibold&quot;;
</string>
           </property>
           <property name="text">
            <string>AM 
PM</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0" colspan="2">
          <widget class="QLabel" name="label_date">
           <property name="minimumSize">
            <size>
             <width>260</width>
             <height>60</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>260</width>
             <height>60</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">color: rgb(255, 255, 255);
font: 600 14pt &quot;Yu Gothic UI Semibold&quot;;
</string>
           </property>
           <property name="text">
            <string>Saturday
99 April 2099</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="label_time">
           <property name="minimumSize">
            <size>
             <width>210</width>
             <height>100</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>210</width>
             <height>100</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>Yu Gothic UI Semibold</family>
             <pointsize>40</pointsize>
             <weight>75</weight>
             <italic>false</italic>
             <bold>true</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">color: rgb(255, 255, 255) ;
font: 600 40pt &quot;Yu Gothic UI Semibold&quot;;
</string>
           </property>
           <property name="text">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot;font: 600 40pt &quot;Yu Gothic UI Semibold&quot;; color:#ffffff;&quot;&gt;99:99&lt;a style=&quot;font-size:18pt;&quot;&gt;.99&lt;/a&gt;&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
           <property name="scaledContents">
            <bool>false</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="1" column="0">
          <widget class="QLabel" name="icon_label">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
             <horstretch>150</horstretch>
             <verstretch>150</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>150</width>
             <height>150</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>150</width>
             <height>150</height>
            </size>
           </property>
           <property name="baseSize">
            <size>
             <width>100</width>
             <height>100</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">color: rgb(255, 255, 255)</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="pixmap">
            <pixmap>Plugins/weather/icons/01n.png</pixmap>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLabel" name="big_weather_label">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="styleSheet">
            <string notr="true">color: rgb(255, 255, 255);
font: 600 40pt &quot;Yu Gothic UI Semibold&quot;;
</string>
           </property>
           <property name="text">
            <string>99 Â°C</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QLabel" name="weather_label">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="styleSheet">
            <string notr="true">color: rgb(255, 255, 255);
font: 350 14pt &quot;Yu Gothic UI Semilight&quot;;

</string>
           </property>
           <property name="text">
            <string>Humidity : 99%
Wind : 99 km/hr</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="descriptive_label">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="styleSheet">
            <string notr="true">color: rgb(255, 255, 255);
font: 350 14pt &quot;Yu Gothic UI Semilight&quot;;
</string>
           </property>
           <property name="text">
            <string>Weather
Location</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QLabel" name="label_activity">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>460</y>
        <width>1371</width>
        <height>71</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">	border: 1px solid;
    border-style: outset;
    padding: 10px;
    background-color: #000000;
    color: #ffffff;
    font-family: &quot;Yu Gothic UI&quot;;
    font-size: 20pt;
/*    font-style: italic; */
    font-weight: bold;
    text-align: center;
    border-radius: 30px;
    border-style: solid; 
    border-color: #333333;
    border-width: 2px;
    padding: 10px;</string>
      </property>
      <property name="text">
       <string>                             Go to the library and find an interesting book</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_1">
     <attribute name="icon">
      <iconset resource="../Data.qrc">
       <normaloff>:/Media/icons/calendar.png</normaloff>
       <normalon>:/Media/icons/calendar.gif</normalon>:/Media/icons/calendar.png</iconset>
     </attribute>
     <attribute name="title">
      <string/>
     </attribute>
     <widget class="QWidget" name="horizontalLayoutWidget">
      <property name="geometry">
       <rect>
        <x>19</x>
        <y>49</y>
        <width>1181</width>
        <height>501</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout" stretch="5,0,5">
       <item>
        <widget class="QListWidget" name="listWidget_today"/>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_refresh_calendar">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="styleSheet">
          <string notr="true"> background-color: #000000;
    color: #FFFFFF;
    border-radius: 10px;
    padding: 10px;</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset>
           <activeoff>:/Media/icons/refresh_64px.png</activeoff>
          </iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>64</width>
           <height>64</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QListWidget" name="listWidget_tomorrow"/>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="icon">
      <iconset resource="../Data.qrc">
       <normaloff>:/Media/icons/smartHome.png</normaloff>
       <normalon>:/Media/icons/smartHome.gif</normalon>:/Media/icons/smartHome.png</iconset>
     </attribute>
     <attribute name="title">
      <string/>
     </attribute>
     <widget class="QTabWidget" name="tabWidget_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>70</y>
        <width>781</width>
        <height>431</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QTabWidget::pane { 
	border-style: solid;
	border-color: #000000;
	border-width: 0 0px 3px 0px; 
}</string>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>My Bedroom</string>
       </attribute>
       <widget class="QPushButton" name="pushButton_led_on">
        <property name="geometry">
         <rect>
          <x>240</x>
          <y>20</y>
          <width>151</width>
          <height>150</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true"> background-color: #333333;
    color: #FFFFFF;
    border-radius: 10px;
    padding: 10px;
icon-size: 128px; /* Set the desired icon size */
    padding-left: 15px; /* Adjust the left padding to center the icon */
    padding-right: 5px; /* Adjust the right padding to center the icon */</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../Data.qrc">
          <normaloff>:/Plugins/smartHome/on_lamp.png</normaloff>:/Plugins/smartHome/on_lamp.png</iconset>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_led_off">
        <property name="geometry">
         <rect>
          <x>70</x>
          <y>20</y>
          <width>151</width>
          <height>150</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true"> background-color: #333333;
    color: #FFFFFF;
    border-radius: 10px;
    padding: 10px;
icon-size: 128px; /* Set the desired icon size */
    padding-left: 15px; /* Adjust the left padding to center the icon */
    padding-right: 5px; /* Adjust the right padding to center the icon */</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../Data.qrc">
          <normaloff>:/Plugins/smartHome/off_lamp.png</normaloff>:/Plugins/smartHome/off_lamp.png</iconset>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_4">
       <attribute name="title">
        <string>My Livingroom</string>
       </attribute>
      </widget>
     </widget>
     <widget class="QPushButton" name="pushButton_fan_on">
      <property name="geometry">
       <rect>
        <x>880</x>
        <y>120</y>
        <width>151</width>
        <height>150</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"> background-color: #333333;
    color: #FFFFFF;
    border-radius: 10px;
    padding: 10px;
icon-size: 128px; /* Set the desired icon size */
    padding-left: 15px; /* Adjust the left padding to center the icon */
    padding-right: 5px; /* Adjust the right padding to center the icon */</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="../Data.qrc">
        <normaloff>:/Media/icons/fan.gif</normaloff>
        <normalon>:/Media/icons/fan.gif</normalon>:/Media/icons/fan.gif</iconset>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_fan_off">
      <property name="geometry">
       <rect>
        <x>710</x>
        <y>120</y>
        <width>151</width>
        <height>150</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"> background-color: #333333;
    color: #FFFFFF;
    border-radius: 10px;
    padding: 10px;
icon-size: 128px; /* Set the desired icon size */
    padding-left: 15px; /* Adjust the left padding to center the icon */
    padding-right: 5px; /* Adjust the right padding to center the icon */</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="../Data.qrc">
        <normaloff>:/Media/icons/fan.png</normaloff>:/Media/icons/fan.png</iconset>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QLabel" name="label_face">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>790</y>
      <width>201</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
font: 600 14pt &quot;Yu Gothic UI Semibold&quot;;
</string>
    </property>
    <property name="text">
     <string>Current Face</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_gesture">
    <property name="geometry">
     <rect>
      <x>1390</x>
      <y>790</y>
      <width>171</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
font: 600 14pt &quot;Yu Gothic UI Semibold&quot;;
</string>
    </property>
    <property name="text">
     <string>Current Gesture</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="../Data.qrc"/>
 </resources>
 <connections/>
</ui>
